src/test/ui/debuginfo/debuginfo_with_uninhabitable_field_and_unsized.rs

```diff
@@ -1,4 +1,4 @@
-// check-pass
+// build-pass
```

Comment:
Turns out we need build-pass to emit the metadata to trigger these tests, oopsies.